Dockerfile

```diff
@@ -52,23 +52,22 @@ ENV PATH $PATH:${ISPC_PATH}
 
 # Install embree
 # https://github.com/embree/embree/releases
-ARG EMBREE_VERSION=2.17.4
+ARG EMBREE_VERSION=3.2.3
 ARG EMBREE_FILE=embree-${EMBREE_VERSION}.x86_64.linux.tar.gz
-
 RUN mkdir -p ${DIST_PATH} \
   && wget https://github.com/embree/embree/releases/download/v${EMBREE_VERSION}/${EMBREE_FILE} \
   && tar zxvf ${EMBREE_FILE} -C ${DIST_PATH} --strip-components=1 \
   && rm -rf ${DIST_PATH}/bin ${DIST_PATH}/doc
 
 # Install OSPRay
 # https://github.com/ospray/ospray/releases
-ARG OSPRAY_VERSION=1.5.0-patch
+ARG OSPRAY_BRANCH=release-1.7.x
```

Comment:
why not using the tag?

Reply:
Because v1.7.0 won't work and they haven't created the v1.7.1 tag yet.